invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/AccessField/components/ProtectionButtons.js

```diff
@@ -76,7 +83,7 @@ export class ProtectionButtons extends Component {
     const { fieldPath } = this.props;
 
     return (
-      <Field
+      <FastField
```

Comment:
why FastField is needed in this case?

Reply:
we needed this so the component is not remounted when we set the initial state of the access field...`Field` components are being remounted apparently whenever you use `setFieldPath` from any component thus it resulted in an indefinite loop...

Reply:
thanks for the explanation, I am still discovering formik quirks...